### ISSUE TITLE
Error thrown from class attribute with trailing whitespace

### ISSUE THREAD

**Issue by dkrasner** [AFASSoftware/maquette#158]
When the class attribute string contains a trailing whitespace the following error is thrown in console: 

```
maquette.umd.js:291 Uncaught DOMException: Failed to execute 'toggle' on 'DOMTokenList': The token provided must not be empty.
```

This looks to be related to a [closed issue](https://github.com/AFASSoftware/maquette/issues/150) but the problem persists. 

[Here](https://codepen.io/anon/pen/BeQmXZ#anon-login) is a codepen example. Changing `textcolor textsize` to `textcolor textsize ` (note the trailing whitespace) will throw the above error in console.

**Comment by johan-gorter** [comment]
Sorry about the late reply (there was an issue with our chat application integration).

The issue is fixed and shipped in [3.3.4](https://github.com/AFASSoftware/maquette/releases/tag/v3.3.4)

**Comment by dkrasner** [comment]
sounds good thank you!
